docs/apv.md

```diff
@@ -349,8 +350,10 @@ Overtreding van lid 1 wordt bestraft met een straf in de 1e categorie.
 
 ### Artikel 41 - Stemherkenning
 
-1. Het is niet toegestaan om een speler te herkennen aan zijn/haar stem als deze zich zodanig heeft gekleed dat hij/zij op uiterlijk onherkenbaar is.
-Het overtreden van het feit beschreven in lid 1 zal worden gestraft met een straf van de eerste categorie.
+1. Het is toegestaan om een speler te herkennen aan zijn/haar stem, tenzij je hem/haar totaal niet kent. 
```

Comment:
Hoe gaan we dit controleren?

Reply:
De persoon dient aan te kunnen tonen dat hij de andere speler kent.